app/assets/javascripts/widgets/emergency-alert.js

```diff
@@ -0,0 +1,76 @@
+var EmergencyAlert = (function() {
+  // Module Vars
+  var emergencyAlertDiv,
+      alertMessage,
+      raveFeed,
+      noEmergencyMessage,
+      isEmergency = false;
+
+  // Module Functions
+  var initialize = function() {
+    emergencyAlertDiv = $('div.emergency-alert');
+    alertMessage = $('div.alert-message').text();
+    // Imposter test feed: https://imposter.chapman.edu/rave.rss
+    raveFeed = 'https://content.getrave.com/rss/chapman/channel1';
+    noEmergencyMessage = "There is currently no emergency"; // current Rave message for no emergency
+
+    // If there is already a message (comes from data def in Cascade),
+    // don't override the HTML with Rave feed
+    if (alertMessage.trim() != '') {
+      displayEmergencyAlert();
+    }
+    else {
+      $.ajax({
+        url: raveFeed,
+        type: 'GET',
+        success: function(data) {
+          $(data).find("item").each(function () {
+            var title = $(this).find("title").text();
+            // Looks for absence of the "no emergency" message
+            isEmergency = title.indexOf(noEmergencyMessage) < 0;
```

Comment:
I really would prefer something like `.includes(noEmergencyMessage)` because it's more readable but we've encountered [browser issues](https://github.com/chapmanu/cascade-assets/issues/305) with that in the recent past.